### DISCUSSION TITLE
How to define the space for opening a new HTML document within the same space on a page

### DISCUSSION THREAD

**Discussion by Y-Zak** [community/community#21454] [Other Feature Feedback, Questions, & Ideas]
<p>With a simple texteditor written page: How to define the space for opening a new HTML document within the same space on a page, and how to retrieve and place it there? That would be very useful. It could be the same style and type of content already defined, kind of having multiple subpages and leaf through. And as a way to open an external menu sheet it would solve the one-menu-for-all-(changing)-pages question of static websites.</p>
<p>see also: <a href="https://www.w3.org/TR/html40/struct/links.html" class="inline-onebox" rel="noopener nofollow ugc">Links in HTML documents</a> under “12.1.3 Specifying anchors and links”</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>I believe you are looking for <code>&lt;iframe&gt;</code> ?</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://www.codingtag.com/how-to-create-nested-webpage-in-html/img/coding-tag-favicon.png" class="site-icon" width="" height="">
      <a href="https://www.codingtag.com/how-to-create-nested-webpage-in-html" target="_blank" rel="noopener nofollow ugc" title="12:00AM - 14 April 2020">codingtag.com – 14 Apr 20</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/361;"><img src="https://user-images.githubusercontent.com/9809727/181064629-9b7f7037-d744-4a65-80a4-4f3ab66e57fe.jpeg" class="thumbnail" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40f8b4c7a050943e5ac5bddef3221d48e0d8fb2d_2_690x361.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40f8b4c7a050943e5ac5bddef3221d48e0d8fb2d_2_1035x541.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/0/40f8b4c7a050943e5ac5bddef3221d48e0d8fb2d.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40f8b4c7a050943e5ac5bddef3221d48e0d8fb2d_2_10x10.png"></div>

<h3><a href="https://www.codingtag.com/how-to-create-nested-webpage-in-html" target="_blank" rel="noopener nofollow ugc">How to create nested webpage in HTML || HTML Elements</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Y-Zak** [comment]
<p>Spot on. Which means one can have for instance a separate navbar document to be updated independently as the site evolves. And which answers many questions about just that. Nice.</p>
